docs/data-integrations/elasticsearch.mdx

```diff
@@ -13,10 +13,10 @@ This handler is implemented using the `elasticsearch` library, the Python Elasti
 
 The required arguments to establish a connection are as follows:
 
-* `hosts` is the host name(s) or IP address(es) of the Elasticsearch server(s). If multiple host name(s) or IP address(es) exist, they should be separated by commas. This parameter is optional, but it should be provided if `cloud_id` is not.
-* `cloud_id` is the unique ID to your hosted Elasticsearch cluster on Elasticsearch Service. This parameter is optional, but it should be provided if `hosts` is not.
-* `username` is the username used to authenticate with the Elasticsearch server. This parameter is optional.
-* `password` is the password used to authenticate the user with the Elasticsearch server. This parameter is optional.
+- `hosts` is the host name(s) or IP address(es) of the Elasticsearch server(s). If multiple host name(s) or IP address(es) exist, they should be separated by commas. This parameter is optional, but it should be provided if `cloud_id` is not.
```

Comment:
You replaced `*` with `-`.
Please change it back to `*`. Thanks.

Reply:
Sorry about that! I had "format on save" turned on in VS Code. I've reverted the dashes to asterisks.